### PR TITLE
docs: rule for transforming filepath to properties

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [ ] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

### CODE REVIEW

File: docs/source/zh-cn/advanced/loader.md [hunk 1]
```diff
@@ -346,19 +357,17 @@ app.loader.loadToApp(directory, 'model', {
 });
 ```
 
-#### lowercaseFirst [Boolean]
-
-是否强制将第一个字符转成小写，默认为 false。
+#### caseStyle [Boolean]
 
-比如加载 `app/controller/User.js`，如果为 true 加载后生成 `app.controller.user`。
+文件的转换规则，可选为 `camel`，`upper`，`lower`，默认为 `camel`。如果指定 `upper` 会将首字母大写，指定 `lower` 会将首字母小写。
```
Comment (dead-horse) [inline docs/source/zh-cn/advanced/loader.md]: 这样写歧义少点吧

```
文件的转换规则可选为 `camel`，`upper`，`lower`，默认为 `camel`，三者都会将文件名转换成驼峰，但是对于首字母的处理有所不同。

- `camel`：首字母不变。
- `upper`：首字母大写。
- `lower`：首字母小写。
```

File: docs/source/zh-cn/core/cookie-and-session.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 title: cookie 与 session
 ---
 
-### cookie
+## cookie
```
Comment (dead-horse) [inline docs/source/zh-cn/core/cookie-and-session.md]: 这个问题已经修复了？

Comment (popomore) [inline docs/source/zh-cn/core/cookie-and-session.md]: 恩，修复了

File: docs/source/zh-cn/advanced/loader.md [hunk 3]
```diff
@@ -346,19 +357,17 @@ app.loader.loadToApp(directory, 'model', {
 });
 ```
 
-#### lowercaseFirst [Boolean]
-
-是否强制将第一个字符转成小写，默认为 false。
+#### caseStyle [Boolean]
```
Comment (fengmk2) [inline docs/source/zh-cn/advanced/loader.md]: 参数类型是string

**Pull request merged**